lib/view/disclaimer_view.dart

```diff
@@ -122,62 +122,65 @@ class _DisclaimerViewState extends State<DisclaimerView> {
                           crossAxisAlignment: CrossAxisAlignment.start,
                           children: <Widget>[
                             Container(
-                              width: 42.0,
+                              margin: const EdgeInsets.only(right: 14.0),
                               child: const Text(
                                 '✋',
                                 style: Styles.textH3,
                               ),
                             ),
-                            const Expanded(
-                              child: Text(
-                                'Please keep in mind\n',
-                                style: Styles.textH3,
-                              ),
-                            ),
+                            Expanded(
+                                child: Column(
```

Comment:
Perhaps flutter format this with dartfmt ?

Reply:
Yup. You are a great detective. Even though I format it manually, it still changes it back to this way.